src/assets/i18n/ru.json5

```
@@ -2000,300 +1986,258 @@
+  // Тут не понимаю, что есть ORE
   // TODO New key - Add a translation
   "collection.edit.tabs.source.form.options.harvestType.METADATA_AND_REF": "Harvest metadata and references to bitstreams (requires ORE support)",
   
   // "collection.edit.tabs.source.form.options.harvestType.METADATA_ONLY": "Harvest metadata only",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.options.harvestType.METADATA_ONLY": "Harvest metadata only",
+  "collection.edit.tabs.source.form.options.harvestType.METADATA_ONLY": "Собирать только метаданные",
   
   // "collection.edit.tabs.source.form.preTransform": "Pre transformation name",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.preTransform": "Pre transformation name",
+  "collection.edit.tabs.source.form.preTransform": "Название до преобразование",
   
   // "collection.edit.tabs.source.form.postTransform": "Post transformation name",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.postTransform": "Post transformation name",
+  "collection.edit.tabs.source.form.postTransform": "Название после преобразование",
   
   // "collection.edit.tabs.source.form.forceSynchronization": "Force Synchronization",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.forceSynchronization": "Force Synchronization",
+  "collection.edit.tabs.source.form.forceSynchronization": "Принудительная Синхронизация",
   
   // "collection.edit.tabs.source.form.recordValidationEnabled": "Record validation",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.recordValidationEnabled": "Record validation",
+  "collection.edit.tabs.source.form.recordValidationEnabled": "Проверка записи",
   
   // "collection.edit.tabs.source.form.itemValidationEnabled": "Item validation",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.itemValidationEnabled": "Item validation",
+  "collection.edit.tabs.source.form.itemValidationEnabled": "Проверка объекта",
   
   // "collection.edit.tabs.source.form.ccAddresses": "CC's addresses",
+  // Что такое CC?
   // TODO New key - Add a translation
   "collection.edit.tabs.source.form.ccAddresses": "CC's addresses",
   
   // "collection.edit.tabs.source.head": "Content Source",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.head": "Content Source",
+  "collection.edit.tabs.source.head": "Источник Контента",
   
   // "collection.edit.tabs.source.notifications.discarded.content": "Your changes were discarded. To reinstate your changes click the 'Undo' button",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.notifications.discarded.content": "Your changes were discarded. To reinstate your changes click the 'Undo' button",
+  "collection.edit.tabs.source.notifications.discarded.content": "Вносимые изменения были отклонены. Для их восстановления нажмите кнопку 'Отменить'",
```

Низкопоклонство перед английскими кавычками!

```suggestion
  "collection.edit.tabs.source.notifications.discarded.content": "Вносимые изменения были отклонены. Для их восстановления нажмите кнопку «Отменить»",
```

---

нормальные кавычки 
а треугольнички русские - бяка